File: src/main/java/com/wikia/tibia/repositories/ObjectRepository.java
package com.wikia.tibia.repositories;

import com.wikia.tibia.gateways.ObjectGateway;
import com.wikia.tibia.objects.TibiaObject;

public class ObjectRepository extends WikiObjectRepository {

    public ObjectRepository() {
        super(TibiaObject.class, new ObjectGateway());
    }
}

File: src/main/kotlin/com/wikia/tibia/repositories/ObjectRepository.kt
package com.wikia.tibia.repositories

import com.wikia.tibia.enums.Contract
import com.wikia.tibia.gateways.WikiObjectGateway
import com.wikia.tibia.objects.TibiaObject

class ObjectRepository : WikiObjectRepository<TibiaObject>(
        wikiObjectClass = TibiaObject::class.java,
        wikiObjectGateway = WikiObjectGateway(Contract.OBJECTS)
)